### PR TITLE
Convert All Alarms Telecare script into a function for processing in phase 1b

### PR DESCRIPTION
**Pull request by Jennit07**
Follows the same format as other functions. First read in data from the SC platform using `read_sc_all_alarms_telecare.R` function and process it using `process_sc_all_alarms_telecare.R`. Some warnings/test failures due to #419

### CODE REVIEW

**Comment by lizihao-anu** [comment]
Removing lazy_dt() did help with the error but it ridiculously increases the function's time consuming (more than 1 hour?). Happy to merge for now and I will keep thinking whether there is a way to increase the efficiency.

**Review by lizihao-anu — APPROVED** [review]

File: R/process_sc_all_alarms_telecare.R [hunk 1]
```diff
@@ -76,8 +76,6 @@ process_sc_all_alarms_telecare <- function(data, sc_demographics = NULL, write_t
 
   # Deal with episodes which have a package across quarters.
   qtr_merge <- at_full_clean %>%
-    # Use lazy_dt() for faster running of code
-    dtplyr::lazy_dt() %>%
```
Comment (lizihao-anu) [inline R/process_sc_all_alarms_telecare.R]: As I checked, using `data.table::as.data.table()` in Package `data.table` can have the similar efficiency to `lazy_dt()` without triggering any issue. But I am not sure if `data.table::as.data.table()` can be recommended here.

**Review by lizihao-anu — APPROVED** [review]
The test run looks good and fast. Happy to merge

**Comment by Jennit07** [comment]
> The test run looks good and fast. Happy to merge

That's great, thanks @lizihao-anu. I will merge this on the training this afternoon as a demo

**Pull request merged**